.github/workflows/stale.yml

```
@@ -0,0 +1,39 @@
+# This workflow warns of PRs that have had no activity for a specified amount of time.
+#
+# For more information, see https://github.com/actions/stale
+name: Mark stale pull requests
+
+on:
+  # Run every day at 00:00 UTC
+  schedule:
+    - cron: '0 0 * * *'
+  # Or run on demand
+  workflow_dispatch:
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v9
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          days-before-pr-stale: 14
+          days-before-issue-stale: -1   # we don't use issues
+          days-before-close: -1         # don't close stale PRs/issues
+          exempt-draft-pr: true         # don't mark draft PRs as stale
+          stale-pr-label: "stale"       # label to use when marking as stale
+
+          stale-pr-message: >
+            This PR has not had activity in the past 2 weeks, labeling it as stale.
```

Maybe shortening "has not had" to "had no" and use "visible" to signal that maybe someone looked but did not (yet) comment?
```suggestion
            This PR had no visible activity in the past 2 weeks, labeling it as stale.
```